gifted_build/src/server/dev-server.js

```diff
@@ -25,6 +29,23 @@ app.use('*', (req, res, next) => {
 
 app.listen(3000, () => console.log('dev app listening on port 3000'))
 
+server.use(bodyParser.urlencoded({extended: false}))
+server.use(bodyParser.json())
 server.use(require('./routes/products'));
 
+server.use(session({
```

Comment:
We should set up a "common_configs" folder that both the dev-server and prod-server can reference.